migrations/V73__Consolidate_Contract_Dates.sql

```diff
@@ -0,0 +1,18 @@
+ALTER TABLE system_intakes ADD COLUMN contract_start_date DATE;
+ALTER TABLE system_intakes ADD COLUMN contract_end_date DATE;
+
+UPDATE system_intakes SET contract_start_date =
```

Comment:
We don't want to migrate existing data because the result will be inaccurate. That's why we have the fallback logic for the display.